plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -439,6 +439,7 @@ private boolean validateRequiredGroups(MissingRequiredProperties validationFailu
     if (!validationFailures.getMissingGroups().isEmpty()) {
       Map.Entry<String, Set<PipelineOptionsProperty>> missingGroupEntry =
           Iterables.getFirst(validationFailures.getMissingGroups().entrySet(), null);
+      Preconditions.checkNotNull(missingGroupEntry); // should be impossible
```

Comment:
Add an error message that there's an inconsistent `MissingRequiredProperties`?

Alternatively (preferably), replace the above with `validationFailures.getMissingGroups().entrySet().iterator().next()`